indexd/index/drivers/alchemy.py

```diff
@@ -878,6 +878,8 @@ def update_blank_record(self, did, rev, size, hashes, urls, authz=None):
 
             record.rev = str(uuid.uuid4())[:8]
 
+            record.updated_date = datetime.datetime.utcnow()
```

Comment:
all other dates in indexd use `.now()` so if you want to use `.utcnow()` maybe we should change it everywhere? 😕

Reply:
oh nvm, i'm wrong, only unit tests use `.now()`

Reply:
that's what it uses when it creates the record in the db :grimacing: so I went based off that

Reply:
at least, it's the default https://github.com/uc-cdis/indexd/blob/500238313f7439c9a15f78533b947c89470fad1a/indexd/index/drivers/alchemy.py#L75